### PR TITLE
Rating & modal

### PR DESCRIPTION
**Pull request by amartya-rs**
feat: rating and modal components
modal
rat

### CODE REVIEW

File: pages/docs.html [hunk 1]
```diff
@@ -1293,6 +1295,62 @@ <h5>Stacked list</h5>
...
+                    </span>
+                    <span class="font-semibold">|</span>
+                    <span class="font-semibold">5</span>
+                </div>
+        
+                </script>
+            </div>
+        
+        </section>
+        <!----------rating-end------------>
```
Comment (iRohitGaur) [inline pages/docs.html:1350]: appreciation: good way to comment code so that we can identify respective sections.

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Review by Shubhamkr2610 — APPROVED** [review]
your code is clean and readable .
appreciation: great that you are using comment line for every  section in beginning and starting

File: pages/docs.html (hunk 1 above)
Comment (amartya-rs) [reply pages/docs.html:1350]: Thanks Rohit!

**Comment by amartya-rs** [comment]
> your code is clean and readable . appreciation: great that you are using comment line for every section in beginning and starting

Thanks Shubham!

**Pull request merged**